config/jobs/kubernetes/sig-k8s-infra/trusted/sig-security-trusted.yaml

```diff
@@ -11,79 +11,21 @@ periodics:
   decorate: true
   extra_refs:
   - org: kubernetes
-    repo: kubernetes
-    base_ref: master
-    path_alias: k8s.io/kubernetes
+    repo: sig-security
```

Comment:
/hold
I trust you all in the abstract, but it's really problematic to have to keep an eye on more sources of arbitrary code in this singular trusted cluster, let's revisit that before making it harder to track down what exactly is running in the trusted cluster. As a general rule we have only allowed:
- launching cloudbuilds with images/builder (which then run in the isolated per-subproject GCP staging project)
- a handful of CI components implemented as jobs

Which are implemented in this repo with a limited approver set.